docs/src/main/sphinx/glossary.rst

```diff
@@ -0,0 +1,204 @@
+========
+Glossary
+========
+
+The glossary contains a list of key Trino terms and definitions.
+
+.. _glossCatalog:
+
+Catalog
+    Catalogs define and name a configuration for connecting to a data source,
+    allowing users to query the connected data. Each catalog's configuration
+    specifies a :ref:`connector <glossConnector>` to define which data source
+    the catalog connects to. For more information about catalogs, see
+    :ref:`trino-concept-catalog`.
+
+.. _glossCA:
+
+Certificate Authority (CA)
+    A trusted organization that examines and validates organizations and their
```

Comment:
I realize this is copied, but it seems confusing for internal CAs. At the core, a CA signs and issues certificates, and is trusted such that its signatures can be used to verify certificates.